gapic-generator/lib/gapic/presenters/resource_presenter.rb

```diff
@@ -75,6 +75,11 @@ def arguments
           @parsed_pattern.arguments
         end
 
+        # @return [String] The conflicting argument name escaped with local binding.
+        def escape_argument arg
+          Gapic::RubyInfo.keywords.include?(arg) ? "binding.local_variable_get :#{arg}" : arg
```

Comment:
I think we'd better use `"binding.local_variable_get(:#{arg})"` (i.e. include parentheses). This is because otherwise the raise will look like

```
raise ::ArgumentError, "case cannot contain /" if binding.local_variable_get :case.to_s.include? "/"
```

which isn't correct: the dot binding is stronger than arguments without parentheses.

Reply:
Fixed it, thanks!